Userland/Libraries/LibDebug/Dwarf/AbbreviationsMap.cpp

```diff
@@ -66,9 +66,13 @@ void AbbreviationsMap::populate_map()
     }
 }
 
-Optional<AbbreviationsMap::AbbreviationEntry> AbbreviationsMap::get(u32 code) const
+Optional<AbbreviationsMap::AbbreviationEntry const*> AbbreviationsMap::get(u32 code) const
```

Comment:
just a ptr maybe?
and on fail just return `nullptr`

also typo in commit message
> LibDebug: Dont copy an AbbreviationEntry [every] time we retrieve a value

Reply:
Sure, thanks!